### PR TITLE
NC News 9 DELETE /api/comments/:comment

### PR DESCRIPTION
**Pull request by ari-abendstern**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -317,8 +311,46 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE:204 deletes the specified comment and sends no body back", () => {
+    return request(app)
+      .delete("/api/comments/7")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .then(({ body: { comments } }) => {
+            expect(comments).not.toContain({
```
Comment (nickkagame) [inline __tests__/app.test.js]: Will always pass as 204 response will never contain anything

**Review by nickkagame — APPROVED** [review]
Nice work Ari!

File: __tests__/app.test.js (hunk 1 above)
Comment (ari-abendstern) [reply __tests__/app.test.js]: Wait have I done it wrong? This was meant to get an array of all the comments and then check that the comment had been successfully deleted

Comment (ari-abendstern) [reply __tests__/app.test.js]: I think I've fixed it!

**Pull request merged**